tensorflow/contrib/cudnn_rnn/python/layers/cudnn_rnn.py

```diff
@@ -30,7 +30,7 @@
 from tensorflow.python.platform import tf_logging as logging
 
 _cudnn_rnn_ops_so = loader.load_op_library(
-    resource_loader.get_path_to_datafile("_cudnn_rnn_ops.so"))
+    resource_loader.get_path_to_datafile("../ops/_cudnn_rnn_ops.so"))
```

Comment:
Can we simply remove this line instead? The earlier line:

```python
from tensorflow.contrib.cudnn_rnn.python.ops import cudnn_rnn_ops
```

imports the `cudnn_rnn_ops` module, which in turn loads the right library anyway, right?

Reply:
Yes I think it'd work. Let's see if it passes the test.

Reply:
@protoget Did it pass on windows?

Reply:
L33 has been removed in the nightly build. It isn't needed since cudnn_rnn.py imports cudnn_rnn_ops which imports the .so file already.

Reply:
Great.

Reply:
Yup, this was done in https://github.com/tensorflow/tensorflow/commit/a6a61884396ef1d51b01f8e13df21becb23fd0c8

which subsumes this PR.

Thanks!